src/core/utils/default_app_categories.ts

```diff
@@ -73,4 +73,40 @@ export const DEFAULT_APP_CATEGORIES: Record<string, AppCategory> = Object.freeze
     order: 5000,
     euiIconType: 'managementApp',
   },
+  // below categories are for workspace
```

Comment:
If the categories are workspace-specific, can we move it to workspace plugin?

Reply:
That depends on how we are going to do navigation changes for plugins. If that's global change, we can keep it here, otherwise it will need move to workspace plugin. Let me move to workspace plugin for this PR.

Reply:
Sounds good to me, we can move it to core at the time when it's needed.

Reply:
moved to workspace plugin.